meeting/meeting_july_6_2021.md

```diff
@@ -0,0 +1,85 @@
+# July 6th (7 PM CST)/6th (9 AM JST) Meeting 2021
+
+## Individual Updates (Sync)
+
+* **#Query(Nadav)**
+* **#Query(Tibo)**
+* **#Query(Lloyd)**
+* **#Query(Chris)**
+* **#Query(Antoine)**
+* **#Query(Jesse)**
+* **#Query(Matt)**
+* **#Query(Ben)**
+* **#Query(Ivan!)**
+* **#Query(all)**
+
+## Implementation Updates (Sync)
+
+* bitcoin-s
+* rust-dlc
+* atomic finance
+* others?
+
+## Specification Writing
+
```

Comment:
Dunno where to fits that but maybe let's have a word about "bLIPs" : https://lists.linuxfoundation.org/pipermail/lightning-dev/2021-June/003086.html, it could make sense to share the DLC optional track (for e.g `premium_fee` for fee-bumping, TLV extensions, ...)